nvflare/private/fed/app/client/worker_process.py

```diff
@@ -105,37 +104,42 @@ def main():
     deployer = None
     command_agent = None
     federated_client = None
+
+    startup = args.startup
```

Comment:
L108 to L122 should have wrap into a function, logging_setup or something.

Reply:
done